x-pack/plugins/apm/public/components/app/service_overview/service_overview_errors_table/index.tsx

```diff
@@ -132,6 +133,7 @@ export function ServiceOverviewErrorsTable({ serviceName }: Props) {
       direction,
       field,
       comparisonType,
+      comparisonEnabled,
```

Comment:
You might want to update the comment above and call out the specific properties that needed to trigger an update even though they are not used

```suggestion
      direction,
      field,
      
      // not used but needed to trigger update when changed
      comparisonType,
      
      // not used but needed to trigger update when changed      
      comparisonEnabled,
```

Reply:
Can you also make it clear why `comparisonType` and `comparisonEnabled` should trigger a new request even though they are not used.

Reply:
@ogupte what does that mean with this PR? sorry I might have missed your point 🤪